src/main/webapp/definitions/alfresco-core.yaml

```
@@ -1297,10 +1297,25 @@ paths:
           }
         }
         ```
-        Any missing aspects are applied automatically. For example, **cm:titled** in the JSON shown above. You can set aspects
+        Any missing aspects are applied automatically. For example, **cm:title** in the JSON shown above. You can set aspects
```

```suggestion
        Any missing aspects are applied automatically. For example, **cm:titled** in the JSON shown above. You can set aspects
```
cm:titled was the correct aspect name that includes the cm:title and cm:description properties

---

Many many thanks :).